packages/next/src/build/webpack-config.ts

```diff
@@ -1442,42 +1434,6 @@ export default async function getBaseWebpackConfig(
         // otherwise NFT will get tripped up
         return `commonjs ${localRes.replace(/.*?next[/\\]dist/, 'next/dist')}`
       }
-      // if the file ends with .shared-runtime, we need to make it point to the correct bundle depending on the layer
-      // this is because each shared-runtime files are unique per bundle, so if you use app-router context in pages,
-      // it'll be a different instance than the one used in the app-router runtime.
-      if (isSharedRuntime) {
-        if (dev) {
-          return `commonjs ${localRes}`
-        }
-
-        const name = path.parse(localRes).name.replace('.shared-runtime', '')
-
-        const camelCaseName = name.replace(/-([a-z])/g, (_, w) =>
-          w.toUpperCase()
-        )
-
-        // there's no externals for API routes but if need be, they'll need to be added here and have
-        // their own layer
-        const runtime =
-          layer === 'app-route-handler'
-            ? 'app-route'
-            : isAppLayer
-            ? 'app-page'
-            : 'pages'
-        return [
-          'commonjs ' +
-            path.posix.join(
-              'next',
-              'dist',
-              'compiled',
-              'next-server',
-              `${runtime}.runtime.${dev ? 'dev' : 'prod'}`
-            ),
-          'default',
-          'sharedModules',
-          camelCaseName,
```

Comment:
Does this removal mean that we are bundling the runtime into the user bundle rather than just referencing the runtime via externals? I don't have an opinion either way as to which was is better but was there a motivation for this change?